proto/osmosis/streamswap/v1/state.proto

```diff
@@ -0,0 +1,113 @@
+// Since: cosmos-sdk 0.43
+syntax = "proto3";
+package osmosis.streamswap.v1;
+
+import "google/protobuf/timestamp.proto";
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v10/x/streamswap/types";
+option (gogoproto.goproto_getters_all) = false;
+
```

Comment:
nit: Can we rename this file to `streamswap.proto`?

Reply:
I think `state.proto` is the most meaningful - it contains types stored in the state. But if you want I can rename.

Reply:
Hrmm I would want to here other reviewer's opinion on this